### ISSUE TITLE
Head position not restoring to default position after death

### ISSUE THREAD

**Issue by PavanLuca** [appgurueu/character_anim#10]
When the character dies the head looks at the last direction while it was still alive.
![Schermata da 2023-11-08 17-10-17](https://github.com/appgurueu/character_anim/assets/123581658/5bd0810f-1df9-4978-bcb8-2f72ab39be2d)

**Comment by appgurueu** [comment]
Is this an issue? Do you want it to look in the "default" direction?

**Comment by PavanLuca** [comment]
Yes, I think it would be more natural, those weird positions when players die are sus 😄.
